solution/ui/regulations/css/scss/partials/_header.scss

```diff
@@ -260,6 +260,7 @@ header {
                                 .header--links__anchor {
                                     @include header__anchor--bold;
 
+                                    color: $mid_blue;
```

Comment:
a minor issue that Jonathan noticed -- these links were the wrong color.  I fixed it here.